akka-docs/src/main/paradox/cluster-client.md

```diff
@@ -227,3 +232,62 @@ This can be useful when initial contacts are provided from some kind of service
 are entirely dynamic and the entire cluster might shut down or crash, be restarted on new addresses. Since the
 client will be stopped in that case a monitoring actor can watch it and upon `Terminate` a new set of initial
 contacts can be fetched and a new cluster client started.
+
+## Migration to Akka gRPC
+
+Cluster Client is deprecated and it is not advised to build new applications with Cluster Client.
+We recommend using [Akka gRPC](https://doc.akka.io/docs/akka-grpc/current/index.html) over using Cluster Client.
```

Comment:
Duplicate of the warning on line 3

Reply:
yes, but this is at the end of the page so deserves repeating